src/upnp.py

```diff
@@ -283,19 +320,24 @@ def sendSearchRouter(self):
             logger.exception("UPnP send query failed")
 
     def createPortMapping(self, router):
-        from debug import logger
+        """Add a port mapping"""
 
         for i in range(50):
             try:
-                routerIP, = unpack('>I', socket.inet_aton(router.address))
+                _, = unpack('>I', socket.inet_aton(router.address))
```

Comment:
If result is not used, do not assign anything.

Reply:
LHS removed - fixed in PR#1270.